docs/docs/examples/guides/04-object-detection-example.md

```diff
@@ -0,0 +1,745 @@
+              hardware: hardware
+    controllers:
+      robot_state_broadcaster:
+        plugin: aica_core_controllers/RobotStateBroadcaster
+        outputs:
+          cartesian_state: /hardware/robot_state_broadcaster/cartesian_state
+        events:
+          transitions:
+            on_load:
+              switch_controllers:
+                hardware: hardware
+                activate: robot_state_broadcaster
+      ik_velocity_controller:
+        plugin: aica_core_controllers/velocity/IKVelocityController
+        inputs:
+          command: /signal_point_attractor/twist
+        events:
+          transitions:
+            on_load:
+              switch_controllers:
+                hardware: hardware
+                activate: ik_velocity_controller
+graph:
+  positions:
+    on_start:
+      x: 300
+      y: 0
+    stop:
+      x: 300
+      y: 100
+    components:
+      yolo_executor:
+        x: 940
+        y: 20
+      yolo_to_marker:
+        x: 1440
+        y: 60
+      signal_point_attractor:
+        x: 1460
+        y: 520
+      cartesian_transformation:
+        x: 960
+        y: 600
+      tf_to_signal:
+        x: 480
+        y: 560
+      video_player_component:
+        x: 500
+        y: 100
+    hardware:
+      hardware:
+        x: 1960
+        y: -20
+  edges:
+    yolo_to_marker_state_command_cartesian_transformation_input_2:
+      path:
+        - x: 1860
+          y: 280
+        - x: 1860
+          y: 580
+        - x: 940
+          y: 580
+        - x: 940
+          y: 860
+    on_start_on_start_video_player_component_video_player_component:
+      path:
+        - x: 480
+          y: 40
+        - x: 480
+          y: 160
+    on_start_on_start_yolo_executor_yolo_executor:
+      path:
+        - x: 680
+          y: 40
+        - x: 680
+          y: 80
+    on_start_on_start_tf_to_signal_tf_to_signal:
+      path:
+        - x: 440
+          y: 40
+        - x: 440
+          y: 620
+    hardware_hardware_robot_state_broadcaster_cartesian_state_signal_point_attractor_state:
+      path:
+        - x: 1400
+          y: 520
+        - x: 1400
+          y: 780
+    yolo_executor_on_activate_yolo_to_marker_yolo_to_marker:
+      path:
+        - x: 1380
+          y: 200
+        - x: 1380
+          y: 120
+    yolo_to_marker_on_activate_signal_point_attractor_signal_point_attractor:
+      path:
+        - x: 1900
+          y: 240
+        - x: 1900
+          y: 540
+        - x: 1420
+          y: 540
+        - x: 1420
+          y: 580
+    tf_to_signal_on_load_cartesian_transformation_cartesian_transformation:
+      path:
+        - x: 900
+          y: 740
+        - x: 900
+          y: 660
+    yolo_to_marker_marker_pose_cartesian_transformation_input_2:
+      path:
+        - x: 1860
+          y: 320
+        - x: 1860
+          y: 500
+        - x: 920
+          y: 500
+        - x: 920
+          y: 860
+```
+
+</details>
+
+## Creating a Video Player Component
```

Comment:
same as above, we could probably offer this component. In fact, this is one of my very old backlog items. We should offer this in our core collection, but I'd prefer to have it in C++ (fyi @domire8)